models/node_helpers.go

```diff
@@ -162,8 +163,14 @@ func createNodeWithID(q *reform.Querier, id string, nodeType NodeType, params *C
 	}
 
 	if params.Region != nil {
-		if err := checkUniqueNodeInstanceRegion(q, params.Address, *params.Region); err != nil {
-			return nil, err
+		if node, err := checkUniqueNodeInstanceRegion(q, params.Address, *params.Region); err != nil {
```

Comment:
We have check for name here https://github.com/percona/pmm-managed/pull/309/files#diff-8907802fa4dd71a9077ac83e9d059b9cR55
Maybe it's better to add check for address there as well?